feature/src/main/java/com/terning/feature/home/home/HomeRoute.kt

```diff
@@ -0,0 +1,118 @@
+package com.terning.feature.home.home
+
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.terning.core.designsystem.component.topappbar.LogoTopAppBar
+import com.terning.core.designsystem.theme.Black
+import com.terning.core.designsystem.theme.Grey150
+import com.terning.core.designsystem.theme.TerningTheme
+import com.terning.core.designsystem.theme.White
+import com.terning.feature.R
+import com.terning.feature.home.home.component.HomeFilteringScreen
+import com.terning.feature.home.home.component.HomeTodayIntern
+
+@Composable
+fun HomeRoute() {
+    HomeScreen()
+}
+
+@OptIn(ExperimentalFoundationApi::class)
+@Composable
+fun HomeScreen() {
+    Scaffold(
+        modifier = Modifier,
+        topBar = {
+            LogoTopAppBar()
+        }
+    ) { paddingValues ->
+        LazyColumn(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(top = paddingValues.calculateTopPadding())
+        ) {
+            item {
+                Column(
+                    modifier = Modifier
+                        .padding(bottom = 16.dp)
+                ) {
+                    Text(
+                        text = stringResource(
+                            id = R.string.home_today_title_start
+                        )
+                                + "남지우"
+                                + stringResource(
+                            id = R.string.home_today_title_end
+                        ),
```

Comment:
```suggestion
                    Text(
                        text = stringResource(id = R.string.home_today_title, "남지우"
                     ),
```
와 같은 방식으로 수정하는게 좋아보여요!! 
문자열 리소스에서 변수를 넣을 부분을 `%s`로 지정하면 됩니다!!

Reply:
와우 진짜 감사합니다!!